pkg/client/install.go

```diff
@@ -107,4 +107,16 @@ spec:
         - containerPort: 44134
           name: tiller
         imagePullPolicy: Always
+				livenessProbe:
```

Comment:
@technosophos fixed in https://github.com/kubernetes/helm/pull/777/commits/9e1df63e7701c1630a4e0f8a7179b70e1b21c174